Sources/ApodiniDatabase/DatabaseModel.swift

```diff
@@ -0,0 +1,15 @@
+import Fluent
+import Vapor
+
+///A protocol all Models that are used with `ApodiniDatabase` need conform to
+public protocol DatabaseModel: Content, Model where IDValue == UUID {
```

Comment:
What is the reason that the `IDValue` is constrained to `UUID`. What would happen if the ID value is something else?

Reply:
This was something I've thought about before we introduced the new Parameter retrieval. The reason was that for `Model.IDValue` needs to conform to `LosslessStringConvertible`, which it does now anyway. So I'll remove that 👍